src/main/studio/core/Studio.java

```diff
@@ -121,79 +121,79 @@ public static void main(final String[] args) {
         SwingUtilities.invokeLater( ()-> init(args) );
     }
 
-    public static boolean hasMacOSSystemMenu() {

```

Comment:
this change was already merged by @gyorokpeter no? in #69

Reply:
the only change was on line 206 `initTaskbarIcon();`
But, what I suspect is that the previous version of the file had CRLF while by IDEA (or git client) replaced CRLF to LF here.

Reply:
indeed looks better after hiding whitespace.